service/role_perm_service.go

```diff
@@ -0,0 +1,69 @@
+package service
+
+import (
+	auth_proto "stlab.itechart-group.com/go/food_delivery/authorization_service/GRPC"
+	"stlab.itechart-group.com/go/food_delivery/authorization_service/model"
+	"stlab.itechart-group.com/go/food_delivery/authorization_service/pkg/logging"
+	"stlab.itechart-group.com/go/food_delivery/authorization_service/repository"
+)
+
+type RolePermService struct {
+	logger logging.Logger
+	repo   repository.Repository
+}
+
+func NewRolePermService(repo repository.Repository, logger logging.Logger) *RolePermService {
+	return &RolePermService{repo: repo, logger: logger}
+}
+
+func (s *RolePermService) GetRoleById(id int) (*model.Role, error) {
+	role, err := s.repo.RolePerm.GetRoleById(id)
+	if err != nil {
```

Comment:
does it make sense to just return what you receive from repo? anyway you don't have any extra logic inside the if-else